server/src/main/java/org/elasticsearch/action/admin/cluster/stats/TransportClusterStatsAction.java

```diff
@@ -320,11 +317,6 @@ public ClusterStatsNodeRequest(StreamInput in) throws IOException {
         public Task createTask(long id, String type, String action, TaskId parentTaskId, Map<String, String> headers) {
             return new CancellableTask(id, type, action, "", parentTaskId, headers);
         }
```

Comment:
Can we remove this override? The effect would be to have it return a plain `Task` instead of a `CancellableTask`. I don't know if that's safe / desirable. (The `createActionContext` method does expect to be passed a `CancellableTask`, if that's relevant.)

Reply:
No, this was added in #68676 as part of making this cancellable